[Style guide] Grammar and Formatting (#9263)
cloudflare/cloudflare-docs: content/style-guide/grammar-and-formatting/numbers-and-units-of-measurement.md

---
pcx_content_type: Concept
title: Numbers and Units of Measurement
---

## Numbers and Units of Measurement

**Be consistent in your use of numbers.** Use numerals for numbers. When you write about numbers used in examples or UI, duplicate them exactly as they appear in the UI. In all other content, follow the guidelines below.

+ In body text, spell out whole numbers from zero through nine.
    + For example: We have added three more servers.
+ For numbers greater than or equal to 10, use digits.
    + For example: The message is 32 bytes long, [...]
+ For analytics, display metrics with numerals.
+ Use commas for numbers with more than 3 digits (no spaces). Note the differences in numerical punctuation for translation.
    + 300
    + 1,000

>     + 7,465

Reviewer: I'm thinking there might be a better way to format this. Can you play around with it (with/without bullets, spacing, etc) and see what works best?

### Using k to indicate thousands

+ Do not put a space between the number and k.
+ Add a noun to indicate what the number measures, and to make clear that you are not using k as an abbreviation for *kilobytes*.

> For example: On this plan, you are limited to 55k download operations and 20k upload operations per day.

Reviewer: should be consistent with previous section. this one is not bulleted, while other is. See comment about playing with formatting

## Units of measurement

Units of measurement can occur in two different contexts.

> ### As nouns: 
> 
> Fast forward to 2020 and the storage provider landscape has become highly competitive with many providers capable of providing **petabyte** (and **exabyte**) scale content storage at extremely low cost-per-**gigabyte**. 	
> 
> ### As modifiers of numbers: 
> 

Reviewer: ```suggestion
### As nouns

Fast forward to 2020 and the storage provider landscape has become highly competitive with many providers capable of providing **petabyte** (and **exabyte**) scale content storage at extremely low cost-per-**gigabyte**. 	

### As modifiers of numbers 

```

---
pcx_content_type: Concept
title: Numbers and Units of Measurement
---

## Numbers and Units of Measurement

**Be consistent in your use of numbers.** Use numerals for numbers. When you write about numbers used in examples or UI, duplicate them exactly as they appear in the UI. In all other content, follow the guidelines below.

+ In body text, spell out whole numbers from zero through nine.

>     + Like, We've added three more servers.

Reviewer: ```suggestion
    + For example: We've added three more servers.
```

+ For numbers >= 10, use digits.

>     + Like, if for example the message is 32 bytes long, [...]

Reviewer: ```suggestion
    + For example: The message is 32 bytes long, [...]
```

> + See UI Content Guidelines for additional information.

Reviewer: add this line to the end of this section instead and link to the page it's referring to (not the wiki link since we don't wanna put that out here)

Author: I think we should leave that line out. I checked the wiki page and it's not going to be published. I can't find any other pages to reference in this folder.

+ For analytics, display metrics with numerals.
+ Use commas for numbers with more than 3 digits (no spaces). Note the differences in numerical punctuation for translation.
    + 300
    + 1,000
    + 7,465

### Using k to indicate thousands

+ Don't put a space between the number and k.

> + Add a noun to indicate what the number measures, and to make clear that you're not using k as an abbreviation for *kilobytes*.

Reviewer: contractions

> Example: On this plan, you are limited to 55k download operations and 20k upload operations per day.

Reviewer: ```suggestion
For example: On this plan, you are limited to 55k download operations and 20k upload operations per day.
```

## Units of measurement

Units of measurement can occur in two different contexts.

> As nouns: 

Reviewer: might be worth making these H3?

> As nouns: 

Reviewer: Might be worth making this h3? Same with other sections like "As modifiers of numbers:", etc.
```suggestion
### As nouns: 
```

Fast forward to 2020 and the storage provider landscape has become highly competitive with many providers capable of providing **petabyte** (and **exabyte**) scale content storage at extremely low cost-per-**gigabyte**. 	

As modifiers of numbers: 

> Many CPU architectures either don't allow execution of the machine code, which is unaligned in memory (**4 KB** for x86 systems)

Reviewer: ```suggestion
Many CPU architectures either do not allow execution of the machine code, which is unaligned in memory (**4 KB** for x86 systems).
```

> In general, it's best to only abbreviate units of measurements when they modify specific quantities/numbers. When you need to abbreviate them:

Reviewer: ```suggestion
In general, only abbreviate units of measurements when they modify specific quantities/numbers. When you need to abbreviate them:
```

+ Always include a space between the number and the unit

>     + ✅10 m
>     + ❌10m
> + Unit symbols should unaltered in the plural.
>     + ✅10 cm
>     + ❌10 cms

Reviewer: ```suggestion
    + ✅ 10 m
    + ❌ 10 m
+ Unit symbols should unaltered in the plural.
    + ✅ 10 cm
    + ❌ 10 cms
```

Author: Agreed, it takes up too much space as it is. How else would you format it? I was thinking we could have fewer examples instead.

Always include a space between the number and the unit
Unit symbols should unaltered in the plural.

✅10 cm
❌10cms

Reviewer: maybe you could format it like the tables that have the correct/incorrect forms? would take up more horizontal space than vertical imo -- and yes to the fewer examples :D
